src/aklite_client_ext.cc

```diff
@@ -239,7 +239,9 @@ bool AkliteClientExt::RebootIfRequired() {
   auto is_reboot_required = client_->isRebootRequired();
 
   if (is_reboot_required.first) {
-    if (is_reboot_required.second.empty()) {
+    if (is_booted_env) {
```

Comment:
I think it should be `if (!is_booted_env)`.

Reply:
Done